### PR TITLE
Add PHP module support to e2e test

### PR DESCRIPTION
**Pull request by heyanlong**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
Add PHP module support to e2e test

### CODE REVIEW

**Comment by wu-sheng** [comment]
I think you never add your e2e test into the CI control file.

**Comment by wu-sheng** [comment]
Ping me about your Docker Hub account, I could add you into the skyapm manager group.

File: test/e2e/e2e-test/docker/php/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    environment:
+      - SW_AGENT_COLLECTOR_BACKEND_SERVICES=oap:11800
+    depends_on:
+      php:
+        condition: service_started
+      oap:
+        condition: service_healthy
+    volumes:
+      - ../php/index.php:/opt/www/index.php
+      - ../php/php-shadow.ini:/usr/local/etc/php/conf.d/ext-skywalking.ini
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/php/docker-compose.yml]:
```suggestion
      - ./index.php:/opt/www/index.php
      - ./php-shadow.ini:/usr/local/etc/php/conf.d/ext-skywalking.ini
```

File: test/e2e/e2e-test/docker/php/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    volumes:
+      - ../php/index.php:/opt/www/index.php
+      - ../php/php.ini:/usr/local/etc/php/conf.d/ext-skywalking.ini
+
+  php-shadow:
+    image: skyapm/skywalking-php:latest
+    networks:
+      e2e:
+        aliases:
+          - php-shadow
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/php/docker-compose.yml]: do we need an alias? if not, let's make it minimal simple

File: test/e2e/e2e-test/docker/php/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    expose:
+      - 8080
+    environment:
+      - SW_AGENT_COLLECTOR_BACKEND_SERVICES=oap:11800
+    depends_on:
+      oap:
+        condition: service_healthy
+    volumes:
+      - ../php/index.php:/opt/www/index.php
+      - ../php/php.ini:/usr/local/etc/php/conf.d/ext-skywalking.ini
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/php/docker-compose.yml]:
```suggestion
      - ./index.php:/opt/www/index.php
      - ./php.ini:/usr/local/etc/php/conf.d/ext-skywalking.ini
```

File: test/e2e/e2e-test/docker/php/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
+  php:
+    image: skyapm/skywalking-php:latest
+    networks:
+      e2e:
+        aliases:
+          - php
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/php/docker-compose.yml]: remove if we don't need it

**Comment by heyanlong** [comment]
@wu-sheng Added in CI control file

File: .github/workflows/e2e.yaml [hunk 5]
```diff
@@ -80,6 +80,28 @@ jobs:
           name: logs
           path: logs
 
+  FeatureGroup03:
```
Comment (wu-sheng) [inline .github/workflows/e2e.yaml]: @kezhenxu94 I have a doubt about the group naming principle. What are they? Why all these say `FeatureGroupxx`, and the last one shows `Gateway`?

Comment (kezhenxu94) [reply .github/workflows/e2e.yaml]: > @kezhenxu94 I have a doubt about the group naming principle. What are they? Why all these say `FeatureGroupxx`, and the last one shows `Gateway`?

The `Gateway` should be also renamed to `FeatureGroupXX` I think, if @wu-sheng you have better suggestion, I'm good to follow :) as for the `PHP` job, as an individual language agent, I prefer to set up another workflow, named `PHP Agent` or `PHP SDK`, WDYT

Comment (wu-sheng) [reply .github/workflows/e2e.yaml]: I prefer it keeps in a separated group, to avoid unexpected CI process conflict with other tests. Also, most PMC know little about the PHP agent, if conflict occurs, it is hard to follow.

@heyanlong Please create `e2e.php.yaml`.
@kezhenxu94 , please rename to `FeatureGroupXX`, at least keep consistent. I don't have better idea for now.

File: test/e2e/e2e-test/docker/php/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+      - e2e
+    restart: on-failure
+    healthcheck:
+      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/11800"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
+  php-shadow:
+    image: skyapm/skywalking-php:latest
```
Comment (wu-sheng) [inline test/e2e/e2e-test/docker/php/docker-compose.yml]: I highly recommended don't use `latest`. Let's image what will happen in the next release? If something breaks in there?
